fcc-test-suite-template-master/index2.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/styles.css" />
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <header id="header">
+      <img id="header-img" src="https://cdn.freecodecamp.org/testable-projects-fcc/images/product-landing-page-logo.png"></img> 
```

Comment:
I would suggest using local images in place of a link.

Reply:
- [ ] You did well here, please make sure you explained a bit more why you suggested the changes. if you give more explanation the student will understand better where the bugs are and how to fix them.